src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -1199,6 +1199,226 @@ Not all possible configuration options may be set via environment variables. The
+        title="AIMonitoring.Streaming.Enabled "
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `true`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_STREAMING_ENABLED`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Configuration function
+          </th>
+
+          <td>
+            `newrelic.ConfigAIMonitoringStreamingEnabled`
+          </td>
+        </tr>
+      </tbody>
+    </table>    
+
+When set to `true`, enables the agent to capture streamed responses. If set to `false`, agent won't capture event data about streamed responses, but the agent can still capture metrics and spans. The span duration will end when the LLM function call exits. When set to `true`, the span duration ends when the final result is read from the stream.
+
+    </Collapser>
+    <Collapser
+        id="ai-monitoring-includeonly"
+        title="AIMonitoring.IncludeOnly"
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            (none)
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_INCLUDE_ONLY`
+          </td>
+        </tr>
+        <tr>
+          <th>
+            Configuration function
+          </th>
+
+          <td>
+            `newrelic.ConfigAIMonitoringIncludeOnly([]string)`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+Directs the agent to collect data from specific libraries. Structured as a comma-separated list. For example:
+
+```go
+newrelic.ConfigAIMonitoringIncludeOnly([]{"openai", "bedrock"}
+```
+
+    </Collapser>
+    <Collapser
+        id="ai-monitoring-record-content"
+        title="AIMonitoring.RecordContent.Enabled"
+    >
+
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `true`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_RECORD_CONTENT_ENABLED`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Configuration function
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_RECORD_CONTENT_ENABLED`
```

Comment:
Configuration function is called `newrelic.ConfigAIMonitoringRecordContentEnabled`